src/components/report-technical-issue/template.njk

```diff
@@ -1,7 +1,7 @@
 {% set language %}{% if params.language == 'cy' %}cy{% else %}en{% endif %}{% endset %}
 
 <a lang="{{ language }}" hreflang="{{ language }}"
-  class="govuk-link {{ params.classList }}"
+  class="govuk-link {{ params.classes }}"
   target="_blank"
-  href="/contact/problem_reports_nonjs?newTab=true{% if params.serviceCode %}&amp;service={{ params.serviceCode }}{% endif %}"
+  href="{{ params.baseUrl }}/contact/problem_reports_nonjs?newTab=true{% if params.serviceCode %}&amp;service={{ params.serviceCode | urlencode }}{% endif %}{% if params.referrerUrl %}&amp;referrerUrl={{ params.referrerUrl | urlencode }}{% endif %}"
 >{% if params.language == 'cy' %}Help gyda'r dudalen hon (yn agor ffenestr neu dab newydd){% else %}Get help with this page (opens in a new tab){% endif %}</a>
```

Comment:
The govuk design system suggest 'opens in new tab' rather than 'opens in a new tab' - https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab. Could you update this whilst you're looking at this component?

Reply:
Updated. 🙂